app/javascript/packages/address-search/index.tsx

```diff
@@ -191,7 +198,7 @@ function AddressSearch({
     handleAddressSearch: onSearch,
     foundAddress,
     validatedFieldRef,
-  } = useUspsLocations();
+  } = useUspsLocations(locationsUrl, addressSearchUrl);
```

Comment:
corresponding switch to keyword args. Also as a style thing in JS we use all caps `URL` (check with `git grep URL -- app/javascript`)
```suggestion
  } = useUspsLocations({ locationsURL, addressSearchURL });
```